src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorConfiguration.cs

```diff
@@ -8,205 +8,268 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace Microsoft.CodeAnalysis.Razor.ProjectSystem.Rules {
```

Comment:
These are generated files that were updated by the new SDK - just drive through 😆